### PR TITLE
Recommend using a service account for BitBucket

### PR DESCRIPTION
**Pull request by runleonarun**
## What are you changing in this pull request and why?

Making sure people understand to use a service account with BitBucket. Some follow up questions:

- Should all git providers use a service account?
 
## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/cloud/git/import-a-project-by-git-url.md [hunk 1]
```diff
@@ -56,7 +56,11 @@ If you use GitLab, you can import your repo directly using [dbt Cloud's GitLab A
 
 ## BitBucket
 
-- To add a deploy key to a BitBucket account, navigate to **SSH keys** tab in the Personal Settings page of your BitBucket account.
+You can import your BitBucket repository into dbt Cloud using a deploy key. To preserve account security, we recommend you use a service account to add the BitBucket deploy key and maintain the connection between your BitBucket repository and dbt Cloud. BitBucket attributes each repository commit and other actions to the actor's Bitbucket email address.
```
Comment (runleonarun) [inline website/docs/docs/cloud/git/import-a-project-by-git-url.md]:
```suggestion
You can import your BitBucket repository into dbt Cloud using a deploy key. To preserve account security, we recommend you use a service account to add the BitBucket deploy key and maintain the connection between your BitBucket repository and dbt Cloud. BitBucket attributes each repository commit and other git actions to the actor's Bitbucket email address.
```

File: website/docs/docs/cloud/git/import-a-project-by-git-url.md [hunk 2]
```diff
@@ -56,7 +56,11 @@ If you use GitLab, you can import your repo directly using [dbt Cloud's GitLab A
 
 ## BitBucket
 
-- To add a deploy key to a BitBucket account, navigate to **SSH keys** tab in the Personal Settings page of your BitBucket account.
+You can import your BitBucket repository into dbt Cloud using a deploy key. To preserve account security, we recommend you use a service account to add the BitBucket deploy key and maintain the connection between your BitBucket repository and dbt Cloud. BitBucket attributes each repository commit and other git actions to the actor's Bitbucket email address.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/git/import-a-project-by-git-url.md]:
```suggestion
Use a deploy key to import your BitBucket repository into dbt Cloud. To preserve account security, use a service account to add the BitBucket deploy key and maintain the connection between your BitBucket repository and dbt Cloud. 

BitBucket links every repository commit and other Git action (such as opening a pull request) to the email associated with the user's Bitbucket account.
```

File: website/docs/docs/cloud/git/import-a-project-by-git-url.md [hunk 3]
```diff
@@ -56,7 +56,13 @@ If you use GitLab, you can import your repo directly using [dbt Cloud's GitLab A
 
 ## BitBucket
 
-- To add a deploy key to a BitBucket account, navigate to **SSH keys** tab in the Personal Settings page of your BitBucket account.
+Use a deploy key to import your BitBucket repository into dbt Cloud. To preserve account security, use a service account to add the BitBucket deploy key and maintain the connection between your BitBucket repository and dbt Cloud. 
+
+BitBucket links every repository commit and other Git action (such as opening a pull request) to the email associated with the user's Bitbucket account.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/git/import-a-project-by-git-url.md]:
```suggestion
BitBucket links every repository commit and other Git actions (such as opening a pull request) to the email associated with the user's Bitbucket account.
```

**Review by mirnawong1 — APPROVED** [review]
🚢

**Pull request merged**